### PR TITLE
Survey results

### PR DESCRIPTION
**Pull request by harshilsharma63**
#### Summary
Added section to display results os previous surveys.

#### Ticket Link
Fixes https://mattermost.atlassian.net/browse/MM-56693

### CODE REVIEW

File: webapp/src/components/common/button/button.tsx
Comment (larkox) [inline webapp/src/components/common/button/button.tsx:1]: :heart:

**Review by larkox — COMMENTED** [review]
LGTM

File: webapp/package-lock.json
Comment (M-ZubairAhmed) [inline webapp/package-lock.json:5855]: what changed this

**Review by M-ZubairAhmed — APPROVED** [review]

File: webapp/src/components/common/button/button.tsx
Comment (harshilsharma63) [reply webapp/src/components/common/button/button.tsx:1]: 👍

File: webapp/src/components/systemConsole/surveyResults/surveyResults.tsx
Comment (harshilsharma63) [reply webapp/src/components/systemConsole/surveyResults/surveyResults.tsx:137]: Thats a great point! Added a basic view for now.

@abhijit-singh LMK if we need to make any changes here. I added this just as a very basic version.

Screenshot 2024-04-08 at 12 43 48 PM

File: webapp/package-lock.json
Comment (harshilsharma63) [reply webapp/package-lock.json:5855]: I might have used incorrect npm version previously and this fix might have been caused by using the right version again. But not sure though.

**Comment by harshilsharma63** [comment]
@larkox I've added the empty view now. UI can be updated whenever later.

**Comment by harshilsharma63** [comment]
Will merge once #6 is approved.

**Pull request merged**
